src/main/resources/com/atlassian/jira/cloud/jenkins/configuration/config.jelly

```diff
@@ -0,0 +1,8 @@
+<?jelly escape-by-default='true'?>
```

Comment:
This is placeholder, the other PR contains the full page code. But i still needed a page to exist so Jenkins didn't break down

Reply:
Ewwww jelly templates